app/menus/menus/help.js

```diff
@@ -1,23 +1,64 @@
-const os = require('os');
+const {release} = require('os');
 const {app, shell} = require('electron');
+const {getConfig, getPlugins} = require('../../config.js');
+
+const appName = app.getName();
+const appVersion = app.getVersion();
 
 module.exports = function (commands, showAbout) {
   const submenu = [
     {
-      label: `${app.getName()} Website`,
+      label: `${appName} Website`,
       click() {
         shell.openExternal('https://hyper.is');
       }
     },
     {
       label: 'Report Issue',
       click() {
+        /* line 23 thru 27 is from app/menus/menu.js
+         need help figuring out how to import; the following wouldn't work:
+         const {updateChannel} = require('../menu'); */
+        const config = getConfig();
+        let updateChannel = 'stable';
+        if (config && config.updateChannel && config.updateChannel === 'canary') {
+          updateChannel = 'canary';
+        }
+
         const body = `
-        <!-- Please succinctly describe your issue and steps to reproduce it. -->
-        -
-        ${app.getName()} ${app.getVersion()}
-        Electron ${process.versions.electron}
-        ${process.platform} ${process.arch} ${os.release()}`;
+<!--
```

Comment:
Can we read this in from the actual template? That way they will always be synced.

Reply:
Or even better, let's change the template in another PR.

it should say something along the lines: "please use the built in error reporting if you have an error, if you can't start hyper, fill in this here:....."

Reply:
I don't know if outputting directly into the template is possible. Maybe create a snippet and wrap those requests in a function for someone to use inside the hyper dev tools console. When I was figuring out how to output the contents I used the hyper devtools console a lot to see what would work. Here are a couple of things I tried. 

this will pop up the file manager
`shell.showItemInFolder(${homedir()}/.hyper.js)`

this will open the .hyper.js file within an editor
`shell.openItem(${homedir()}/.hyper.js)`

this will log the hyper contents to a global var inside hyper and logs everything to the console beside  `${app.getName()} ${app.getVersion()}` <- I couldn't figure out how to print these to the console
```
readFile(`${homedir()}/.hyper.js`, 'utf-8', (err, config)=>{
   if (err) throw "err";
   console.log(global.hyperjs = config,
    `"\nElectron", ${process.versions.electron}`,
 `${process.platform}, ${process.arch}, ${release()}`)
   }
)
```

I wanted to suggest some changes for the current issue template too and additionally I wanted to suggest a CONTRIBUTING.md under .github/ because a [Please review the guidelines for contributing to this repository](https://help.github.com/articles/setting-guidelines-for-repository-contributors/) banner/link will be displayed whenever someone opens a pull request or creates an issue.

That was something I was interested in doing but I was waiting for this pr to be done.

Reply:
I think the way it's implemented now is good 👍  let's tweak it later